web/src/admin/AdminInterface/AdminSidebar.ts

```diff
@@ -98,117 +138,96 @@ export class AkAdminSidebar extends AKElement {
         this.classList.add(this.open ? "pf-m-expanded" : "pf-m-collapsed");
     }
 
-    renderSidebarItems(): TemplateResult {
-        // The second attribute type is of string[] to help with the 'activeWhen' control, which was
-        // commonplace and singular enough to merit its own handler.
-        type SidebarEntry = [
-            path: string | null,
-            label: string,
-            attributes?: Record<string, any> | string[] | null, // eslint-disable-line
-            children?: SidebarEntry[],
-        ];
+    get sidebarItems(): SidebarEntry[] {
+        const reload = () =>
+            new CoreApi(DEFAULT_CONFIG).coreUsersImpersonateEndRetrieve().then(() => {
+                window.location.reload();
+            });
+
+        const newVersionMessage: LocalSidebarEntry[] =
```

Comment:
None of the "special cases" were all that special.  They all conformed to some navigational type, so they didn't have to be marked in any way, other than being part of the tree.